shared/cmd/flags.go

```diff
@@ -53,6 +53,11 @@ var (
 		Name:  "no-discovery",
 		Usage: "Enable only local network p2p and do not connect to cloud bootstrap nodes.",
 	}
+	// StaticPeers specifies a set of peers to connect to explicitly.
+	StaticPeers = cli.StringSliceFlag{
+		Name:  "peer",
```

Comment:
Does this accept a comma separated list of peers? 

Please update the name and/or description. Reading this makes me think I would write `--peer=hostA:1234,hostB:1234,hostC:1234` but i'm not sure if it is supposed to `--peer=hostA:1234 --peer=hostB:1234 --peer=hostC:1234`

Reply:
I tried it manually. Indeed right now it's the latter: `--peer=hostA:1234` `--peer=hostB:1234` `--peer=hostC:1234`. Would you rather like to use a comma separated entry? Please advise. We may be able to combine both possibilities.

Reply:
OK great I understand. This is OK

Reply:
I have a patch for you. Just a sec.